src/main/java/org/purpurmc/purpurextras/modules/PurpurExtrasModule.java

```diff
@@ -22,6 +24,11 @@ public interface PurpurExtrasModule {
      */
     boolean shouldEnable();
 
+    default boolean isRegistered(Permission permission) {
```

Comment:
I'd named that isPermissionRegistered so I dont't need to look up what it's checking

Reply:
renamed